### PR TITLE
feat: --location becomes --geographic-description TDE-985

### PR DESCRIPTION
**Pull request by paulfouquet**
#### Motivation

The `standardise_validate.py` `--location` argument becomes `--geographic-description` (https://github.com/linz/topo-imagery/pull/765).

#### Modification

- Update the `standardise-validate` task to take account of the parameter change.
- Upgrade default version of the `topo-imagery` container.

#### Checklist

- [ ] Tests updated - N/A
- [x] Docs updated
- [x] Issue linked in Title

### CODE REVIEW

File: workflows/raster/README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ In addition, a Basemaps link is produced enabling visual QA.
 | licensor_list          | str   |                                       | List of imagery licensors, separated by a semicolons (;). :warning: Has no effect unless a semicolon delimited list is entered.                         |
 | start_datetime         | str   | YYYY-MM-DD                            | Imagery start date (flown from), must be in default formatting                                                                                          |
 | end_datetime           | str   | YYYY-MM-DD                            | Imagery end date (flown to), must be in default formatting                                                                                              |
-| location               | str   | Hamilton                              | (Optional) Additional datatset description, to be used in dataset title / description in place of the Region.                                           |
+| geographic_description | str   | Hamilton                              | (Optional) Additional datatset description, to be used in dataset title / description in place of the Region.                                           |
```
Comment (l0b0) [inline workflows/raster/README.md:39]:
```suggestion
| geographic_description | str   | Hamilton                              | (Optional) Additional dataset description, to be used in dataset title / description in place of the Region.                                           |
```

Also, it looks like a bunch of the "defaults" in the third column are either examples or format strings, and should be in the description instead.

File: workflows/raster/README.md [hunk 2]
```diff
@@ -291,7 +291,7 @@ This workflow carries out the steps in the [Standardising](#Standardising) workf
...
 | source_epsg            | str   | 2193         | The EPSG code of the source imagery.                                                                                                                                                                                                             |
 | target_epsg            | str   | 2193         | The Target EPSG code, if different to source-epsg the imagery will be reprojected.                                                                                                                                                               |
-| location               | str   | ''           | (Optional) Additional datatset description, to be used in dataset title / description in place of the Region.                                                                                                                                    |
+| geographic_description | str   | ''           | (Optional) Additional datatset description, to be used in dataset title / description in place of the Region.                                                                                                                                    |
```
Comment (l0b0) [inline workflows/raster/README.md:294]:
```suggestion
| geographic_description | str   | ''           | (Optional) Additional dataset description, to be used in dataset title / description in place of the Region.                                                                                                                                    |
```

File: workflows/raster/README.md [hunk 3]
```diff
@@ -15,34 +15,66 @@ In addition, a Basemaps link is produced enabling visual QA.
 
 ## Workflow Input Parameters
 
-| Parameter              | Type  | Default                               | Description                                                                                                                                                                                                           |
+<<<<<<< HEAD
```
Comment (l0b0) [inline workflows/raster/README.md:18]: This is a Git conflict marker, and shouldn't be here.

Comment (paulfouquet) [reply workflows/raster/README.md:18]: file has not been saved...

File: workflows/raster/README.md [hunk 4]
```diff
@@ -15,34 +15,66 @@ In addition, a Basemaps link is produced enabling visual QA.
...
+| licensor_list | str | | List of imagery licensors, separated by a semicolons (;). :warning: Has no effect unless a semicolon delimited list is entered. |
+| start_datetime | str | YYYY-MM-DD | Imagery start date (flown from), must be in default formatting |
+| end_datetime | str | YYYY-MM-DD | Imagery end date (flown to), must be in default formatting |
+| geographic_description | str | Hamilton | (Optional) Additional dataset description, to be used in dataset title / description in place of the Region. |
+| lifeycle | enum | Completed | Lifecycle Status of Collection |
+| event | str | Cyclone Gabrielle | (Optional) Event name if dataset has been captured in association with an event. |
+| historic-survey-number | str | SNC8844 | (Optional) Survey Number associated with historical datasets. |
+| copy_option | enum | --no-clobber | `--no-clobber` Skip overwriting existing files. `--force` Overwrite all files. `--force-no-clobber` Overwrite only changed files, skip unchanged files. |
+| source_epsg | str | 2193 | The EPSG code of the source imagery |
+| target_epsg | str | 2193 | The target EPSG code - if different to source the imagery will be reprojected |
```
Comment (l0b0) [inline workflows/raster/README.md]: I'm starting to feel like a `man`-page style would be better at this point. So this table would be formatted something like this (adapted from `man grep`):

```text
PARAMETERS
   [section name; optional grouping of parameters]
       [parameter name]
              [parameter description]
```

Example:

```text
PARAMETERS
   Producers
       producer
              List of imagery producers, separated by a semicolons (;). :warning: Has no effect unless a semicolon delimited list is entered.
```

**Review by l0b0 — APPROVED** [review]

**Review by MDavidson17 — APPROVED** [review]

**Pull request merged**